<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Control</class>
 <widget class="QWidget" name="Control">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>319</width>
    <height>150</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="Control_2" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>301</width>
     <height>131</height>
    </rect>
   </property>
   <widget class="QPushButton" name="paraFrente">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>97</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Para Frente</string>
    </property>
   </widget>
   <widget class="QPushButton" name="paraTraz">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>97</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Para Traz</string>
    </property>
   </widget>
   <widget class="QPushButton" name="giraDireita">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>50</y>
      <width>97</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Gira Direita</string>
    </property>
   </widget>
   <widget class="QPushButton" name="giraEsquerda">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>97</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Gira Esquerda</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
